libs/providers/flagd-web/README.md

```diff
@@ -0,0 +1,17 @@
+# Client-side flagD Provider for OpenFeature
+
+![Experimental](https://img.shields.io/badge/experimental-breaking%20changes%20allowed-yellow)
+
+## Installation
+
+```
+$ npm install @openfeature/flagd-web-provider
+```
+
+## Building
```

Comment:
Add section on the git submodule and Buf requirements. I believe you already have something similar in the server-side provider.